modules/installation/proc-installing-with-helm-cli.adoc

```diff
@@ -0,0 +1,56 @@
+[id="installing-with-helm-cli_{context}"]
+= Installing {product} with Helm CLI
+
+You can use the Helm CLI to install {product} on OpenShift or Kubernetes.
+
+.Prerequisites
+* An `oc` or `kubectl` session with a user that has the `developer` or `admin` role on an OpenShift or Kubernetes cluster.
+* Helm CLI.
+
+.Procedure
+. For a Kubernetes cluster, you  must first add the OpenShift Helm Charts repository. On Openshift, this is already present. 
++
+----
+$ helm repo add openshift-helm-charts https://charts.openshift.io/
+----
+
+. Create and activate the _<rhdh>_ OpenShift project or Kubernetes namespace:
++
+[subs="quotes+"]
+----
```

Comment:
```suggestion
[source,terminal]
[subs="quotes+"]
----
```

Add the tags for proper syntax highlighting to code blocks generally - not positive how this works using the "subs" or why we need those?

Reply:
AFAIK, we use these throughout RHDH docs from what I've seen, so it's consistent with the rest of our docs